uavcan/equipment/device/1100.Temperature.uavcan

```diff
@@ -0,0 +1,11 @@
+#
+# Generic device temperature
+#
+
+uint16_t device_id
+
+float16 temperature
```

Comment:
While we're at it, it would help to add a comment indicating that the temperature should be in Kelvin.